src/components/infoCard/ContactSection.tsx

```diff
@@ -45,7 +49,9 @@ const ContactView = ({ contact }: { contact?: Contact }) => {
           <TouchableOpacity onPress={() => openLink(`mailto:${contact.email}`)}>
             <RegularText
               primary
-              accessibilityLabel={`(E-Mail) ${contact.email} (Taste) (Wechselt zur E-Mail-App)`}
+              accessibilityLabel={
+                (a11yText.mail, `${contact.email}`, a11yText.button, a11yText.mailHint)
+              }
```

Comment:
it is not a whole string anymore which we need here

Reply:
done in d906981aa68e8a03252497b159ecff3df02be64f